tools/docs/test/index.test.mjs

```diff
@@ -0,0 +1,78 @@
+import test from 'node:test';
+import fs from 'fs-extra';
+import { glob } from 'glob';
+import assert from 'node:assert';
+import MarkdownIt from 'markdown-it';
+import remark from 'remark';
+import path from 'node:path';
+
+const markdown = new MarkdownIt('zero');
+markdown.enable(['fence']);
+
+const root = path.resolve('tmp/docs');
+
+/**
+ * @param {any} node
+ * @param {Set<string>} files
+ * @param {string} file
+ */
+function checkNode(node, files, file) {
+  if (node.type === 'link') {
+    /** @type {import('mdast').Link} */
+    const link = node;
+    assert.ok(
+      !link.url.startsWith('/'),
+      'Link should be external or relative: ' + link.url,
+    );
+
+    if (link.url.startsWith('.')) {
+      const abs = path.resolve(
```

Comment:
Is it clear to you what `const abs` means? Just reading the name doesn't help me.

Reply:
will rename